roles/cs.php-tideways/templates/tideways-mageops.ini

```diff
@@ -0,0 +1,10 @@
+tideways.api_key = {{ tideways_api_key }}
+tideways.service = {{ tideways_service }}
+tideways.framework = {{ tideways_framework }}
+tideways.sample_rate = {{ tideways_sample_rate }}
+tideways.enable_cli = {{ tideways_enable_cli | ternary('1', '0') }}
+tideways.auto_start = {{ tideways_auto_start | ternary('1', '0') }}
+tideways.sql_long_query_time_ms = {{ tideways_sql_long_query_time_ms }}
+tideways.traces_only_keep_minimum_ms = {{ tideways_traces_only_keep_minimum_ms }}
+tideways.monitor = {{ tideways_monitor_mode }}
+tideways.collect = {{ tideways_collect_mode }}
```

Comment:
`tideways.dynamic_tracepoints.enable_web` is a setting that should be included here, set to 0 by dfeault and allowed to be enabled by the user. Its a central feature that is opt-in: https://support.tideways.com/documentation/setup/configuration/tracepoints.html#configuration-for-dynamic-tracepoints

Reply:
Thanks for the hint! Will be corrected :)

Since I have you here - I've got a small question concerning CLI tracing - I've seen transactions originating from (completely unrelated to the application) CLI helper running from system cron even though I had `tideways.enable_cli  = false`. It might have been a leftover from some partial configuration during testing, however, this makes me wonder if this setting has the intended effect.